src/replication_misc.c

```diff
@@ -25,25 +24,31 @@ replication_connect(const char *host, int port)
 	int val = 1;
 	int rc;
 
-	if (host == NULL) {
+	if (host == NULL)
+	{
```

Comment:
these changes are not required, and deviates from the other code in coding style.
And, these are never part of cstyle errors of this file.